libs/editor/WordPressEditor/src/main/res/values/colors.xml

```diff
@@ -17,9 +17,9 @@
     <color name="bullet">@color/wp_grey_darken_30</color>
     <color name="code">@color/text</color>
     <color name="code_background">@color/wp_grey_light</color>
-    <color name="hint_text">@color/wp_grey</color>
+    <color name="hint_text">@color/wp_grey_darken_10</color>
     <color name="link">@color/wp_blue_medium</color>
-    <color name="quote">@color/wp_grey</color>
+    <color name="quote">@color/wp_grey_darken_10</color>
```

Comment:
```suggestion
    <color name="quote">@color/wp_grey</color>
```

We don't need to change the value for the `quote` color in order to update the hint text.  The `quote` color is used for the vertical line to the left of a quote.  See the screenshots below for illustration.

![11027](https://user-images.githubusercontent.com/3827611/71917939-7fcb8880-313e-11ea-85b1-9ac8d208721d.png)

Reply:
Thanks! I reviewed the diff in GitKraken and didn't see this quote change. Must have slipped in somehow. Thanks for the review!